docs/Conformance-Guide-draft.md

```diff
@@ -0,0 +1,103 @@
+
+1. (Required if VS Code) VS Code extension dependency: **Zowe Explorer VS Code extensions can only be activated after Zowe Explorer is fully activated itself. Therefore, to ensure the correct activation order and extender must include an extensionDependencies to Zowe Explorer entry in their package.json file.**
+
+   The package.json file must be provided to verify this criteria.
+
+   ```json
+   "extensionDependencies": [
+    "Zowe.vscode-extension-for-zowe"
+   ],
+   ```
+
+1. (Required if VS Code) Zowe Extender access: **Zowe Explorer VS Code extensions must access the shared Zowe Explorer profiles cache only via the `ZoweExplorerApi.IApiRegisterClient.getExplorerExtenderApi()` API as documented in the Zowe Explorer extensibility documentation.**
+
+   API calls to the Explorer Extender API will be logged by Zowe Explorer and be verified by inspecting Zowe Explorer log files.
+
+1. (Required if VS Code) Added Profile Type initialization: **If the extender depends on a new Zowe CLI profile type other than the Zowe Explorer default `zosmf` then an extender needs to call `ZoweExplorerApi.IApiRegisterClient.getExplorerExtenderApi().initialize(profileTypeName)`**
+
+   **(TBD)** as its first API call to ensure that the profile type can be supported. This call ensures that the profiles of this type can be successfully managed even without the Zowe CLI plugin being installed locally for pure VS Code users as well as that the profiles can successfully be accessed in the secure credentials store. This registration call will be logged by Zowe Explorer and be verified by inspecting Zowe Explorer log files.
+
+## Data Provider extension conformance criteria (2)
+
+A data provider Zowe Explorer extension provides an alternative protocol for Zowe Explorer to interact with z/OS. The default protocol Zowe Explorer uses is the z/OSMF REST APIs and data provider adds support for another API. For example, the Zowe Explorer extension for zFTP, which is maintained by the Zowe Explorer squad is an example for a Zowe Explorer data provider extension that uses FTP instead of z/OSMF for all of its USS and MVS interactions. To achieve such an extension it uses a Zowe CLI Plugin for FTP that implemented the core interactions and provided them as an SDK andd
+
+1. (Required) New Zowe CLI profile type: **A new data provider requires a new Zowe CLI profile type as it is the foundation for registering the APIs that add the new protocol as documented in the extension guideline. When new APIs get registered via the `ZoweExplorerApi.IApiRegisterClient.registerMvsApi()` call the APIs passed as a parameter needs to use a unique profile type name.**
```

Comment:
The extension registers its new API via the ZoweExplorerApi.IApiRegisterClient.registerMvsApi() call as indicated from the Zowe Explorer extensibility documentation

Reply:
Building on @jellypuno's suggestion: Move rationale to the paragraph below, rather than removing it completely.

Reply:
I think we need to point out that the registration of new APIs work with a newly defined profile type, i.e. you cannot register an alternative zosmf implementation, for example.

Reply:
Extension registers its new API instances with a new profile type name for the different Zowe Explorer views via the `ZoweExplorerApi.IApiRegisterClient.register{Mvs|Uss|Jes}Api(profileTypeName)` call as indicated from the Zowe Explorer extensibility documentation